packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -527,10 +527,14 @@ export default class TradeStore extends BaseStore {
     };
 
     @action.bound
-    onPurchase(proposal_id, price, type) {
+    onPurchase = debounce(this.onPurchaseRaw, 300);
```

Comment:
Could we improve naming? We have other `onPurchase...` functions here so it's hard to say which is which.

Reply:
@salar-fs yeah actually I faced this issue, but couldn't find a better convenient naming, as we use `onPurchase` in the project, so changing it would cause a conflict in developers mind. so I changed the raw version to `onPurchaseRaw`. I would appreciate if you can suggest a better naming